packages/roosterjs-editor-api/lib/test/format/clearFormatTest.ts

```diff
@@ -37,6 +37,6 @@ describe('clearFormat()', () => {
         clearFormat(editor);
 
         // Assert
-        expect(editor.getContent()).toBe('<div id="text" style="">text</div>');
+        expect(editor.getContent()).toBe('<div id="text" style=""><span style="font-family: &quot;times new roman&quot;; font-size: 16px; color: black; background-color: rgba(0, 0, 0, 0);">text</span></div>');
```

Comment:
Based on this test, shouldn't all the formatting be removed? It looks like it partially shifted to the interior span element, but those styles don't match.

Reply:
Ideally, yes, we should remove those styles.
But it is difficult to make it work correctly, especially when there are many nested elements.
So the dev code is to apply default format to the content. This results in a lot of styles are set into the elements.

I was thinking about having a parameter to clearFormat() to allow switching apply default format. But since in some browsers (IE, Edge), it will not work correctly without this action, it doesn't make sense to me to skip this action and leave a not well-cleared html content. So I decide to always clear these formats by applying default format, and allow it has these styles on.

Reply:
That makes sense. Is there a reason for the addition of a span instead of applying these styles to the div? I suppose to handle the case of a partial selection being cleared, but should that be tested?

Just a thought.